CHANGELOG.md

```diff
@@ -6,6 +6,29 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+- more [Task Enum](./README.md#task-enum) tasks
+- [accelerator](./README#accelerators) options in [Runtime Object](./README#runtime-object)
+- [Model Output Object](./README.md#model-output-object)
+- batch_size and hardware summary
+- [`disk_size`, `memory_size`](./README#architecture-object)
+- [`hardware_summary`, `accelerator`, `accelerator_constrained`](./README#runtime-object) to specify hardware requirements for inference
+- Use common metadata [Asset Object](https://github.com/radiantearth/stac-spec/blob/master/collection-spec/collection-spec.md#asset-object) to refer to model asset and source code.
+- flexible [class map object](./README.md#class-map-object) and [parameters object](./README.md#parameters-object) to handle aspects of models that vary substantially in number
+
+### Changed
```

Comment:
Please add an entry that describes the change of `dlm:architecture` nested fields to corresponding `mlm:name`, `mlm:summary` and so on to provide the STAC API search capabilities.